doc/INTEGRATION.md

```diff
@@ -0,0 +1,89 @@
+
+[Multisig Prover Documentation](../doc/src/contracts/multisig_prover.md)
+
+
+### ampd
+
+Ampd is the off chain daemon responsible for voting and signing within the amplifier protocol. The daemon listens to the Axelar blockchain for specific events, connects to external blockchains via JSON-RPC, and uses [tofnd](https://github.com/axelarnetwork/tofnd) for signing transactions and batches of messages
+
+
+### Infrastructure Contracts
+
+These core contracts manage essential infrastructure logic for the protocol and typically do not require modification. If your chain does require the modificaiton of any of these contracts, please let us know by filing a Github issue to this repository to create a pull request to be merged. 
+
+**Coordinator**  
+
+Manages coordination between contracts in the Amplifier system. [Coordinator Documentation](../doc/src/contracts/coordinator.md)
+
+**Multisig**  
+
+Facilitates verifier signing sessions that is leveraged by the multisig prover contract. The Multisig contract currently supports ECDSA and ED25519 signature schemes. [Multisig Documentation](../doc/src/contracts/multisig.md)
+
+**Rewards**  
+
+Manages verifier rewards. [Rewards Documentation](../doc/src/contracts/rewards.md)
+
+**Router**  
+
+Routes messages between gateway contracts connected in the Amplifier ecosystem, and responsible for performing various administrative functions for chain. [Router Documentation](../doc/src/contracts/router.md)
+
+**Service Registry**  
+
+Registers and tracks the pool of verifiers that support each and every Amplifier chain. [Service Registry Documentation](../doc/src/contracts/service_registry.md)
+
+## Integration Steps
+
+### 1. Deploy and instantiate the chain-specific contracts
+
+The very first step is to deploy and instantiate the chain-specific contracts. This repository has reference contracts for each: [**Gateway**](../contracts/gateway), [**Voting Verifier**](../contracts/voting-verifier), and [**Multisig Prover**](../contracts/multisig-prover), and these reference contracts have been used in many of the initial chains live on Amplifier today. 
+
+However, it is up to the integrator to determine how to best utilize the reference contracts, whether that may be by using the reference contracts we expose as is, maintaining a fork, or something else. It is important to note that these reference contracts are under active development and may change at any time, so integrators should be aware of the specific implementation and version of any reference contracts should they decide to use them.
+
+### 2. Configure `ampd`
+
+If your chain is **not EVM-compatible**, you will need to implement an **ampd** module that Amplifier can use to communicate with your chain. Using the [ampd EVM module](../ampd/src/evm) as a design template, you will have to fork this repository and create a pull request to merge into the main repository.
```

Comment:
Just to be super clear, add a note about how any changes to ampd need to be merged into the main repository, which differs from any changes made to the chain specific contracts (where they will just maintain their own fork).